build-system/pr-check/unminified-tests.js

```diff
@@ -51,7 +50,7 @@ function main() {
         'Integration tests failed! Skipping remaining tests.'
       );
       timedExecOrThrow(
-        'gulp unit --nobuild --headless --coverage --report',
+        'gulp unit --headless --coverage --report',
```

Comment:
Why this change?

Reply:
Oh, I meant to do this in the next PR (coming up shortly). Since `--nobuild` is effectively a no-op for unit tests, so I'm going to leave this in for now.